integrated_channels/api/v1/sap_success_factors/views.py

```diff
@@ -0,0 +1,40 @@
+"""
+    Viewset for integrated_channels/v1/sap_success_factors/
+"""
+from edx_rbac.mixins import PermissionRequiredMixin
+from edx_rest_framework_extensions.auth.jwt.authentication import JwtAuthentication
+from rest_framework import permissions, viewsets
+from rest_framework.authentication import SessionAuthentication
+
+from django.core.exceptions import PermissionDenied
+from django.http import Http404
+
+from enterprise.models import EnterpriseCustomerUser
+from integrated_channels.sap_success_factors.models import SAPSuccessFactorsEnterpriseCustomerConfiguration
+
+from .serializers import SAPSuccessFactorsConfigSerializer
+
+
+class SAPSuccessFactorsConfigurationViewSet(PermissionRequiredMixin, viewsets.ModelViewSet):
+    authentication_classes = (JwtAuthentication, SessionAuthentication,)
+    permission_classes = (permissions.IsAuthenticated,)
+    serializer_class = SAPSuccessFactorsConfigSerializer
+    permission_required = 'enterprise.can_access_admin_dashboard'
+
+    def get_queryset(self):
+        enterprise_customer_users = EnterpriseCustomerUser.objects.filter(user_id=self.request.user.id)
+        if enterprise_customer_users:
+            enterprise_customers_uuids = [str(enterprise_customer_user.enterprise_customer.uuid) for
```

Comment:
Wait, I'm confused, how can an enterprise customer have multiple UUIDs?

Reply:
an enterprise customer (ie just an enterprise) would not have multiple uuids.
An enterprise customer _user_  can belong to multiple enterprises (see ECS, devs testing things, etc.). It would only affect those of us internal more than likely though.

This code is taking the user id and finding the enterprise UUIDs that the user is associated to, which can be plural.